diff --git a/dist/drawer/Drawer.svelte b/dist/drawer/Drawer.svelte
index 51c2bfdf6141f65ee8c5b414e231bfe3c1cc1e56..e57a0fd8c1200e10cc36dd26f37d0335b273427a 100644
--- a/dist/drawer/Drawer.svelte
+++ b/dist/drawer/Drawer.svelte
@@ -53,7 +53,7 @@ let backdropDivClass = twMerge('fixed top-0 left-0 z-50 w-full h-full', backdrop
       {id}
       {...$$restProps}
       class={twMerge(divClass, width, position, placements[placement], $$props.class)}
-      transition:multiple={transitionParams}
+      transition:multiple|global={transitionParams}
       tabindex="-1"
       aria-controls={id}
       aria-labelledby={id}>
@@ -64,7 +64,7 @@ let backdropDivClass = twMerge('fixed top-0 left-0 z-50 w-full h-full', backdrop
       {id}
       {...$$restProps}
       class={twMerge(divClass, width, position, placements[placement], $$props.class)}
-      transition:multiple={transitionParams}
+      transition:multiple|global={transitionParams}
       tabindex="-1"
       aria-controls={id}
       aria-labelledby={id}>